src/index.ts

```diff
@@ -133,5 +135,7 @@ export {
     usePoolTokenNeedsApproval,
     usePoolTokenApproval,
     AllowanceError,
-    TransactionError
+    TransactionError,
+    swap,
+    calculateSwap
```

Comment:
We should create Types around the Swap contract here & the functions we expose to interact with them (so for now, calculateSwap & swap, also buildSwap when implemented(maybe with different naming))